Documentation/Metrics/arangodb_arangosearch_num_docs.yaml

```diff
@@ -0,0 +1,13 @@
+name: arangodb_arangosearch_num_docs
+introducedIn: "3.10.0"
+help: |
+  Number of documents for current snapshot.
+unit: number
+type: gauge
+category: Disk
```

Comment:
Does it make sense to have a separate category for search?

Reply:
I think yes. @gnusi?

If we want to add it, can I add it in the next pr with additional tests?